awscli/customizations/wizard/ui/keybindings.py

```diff
@@ -22,20 +25,26 @@ def exit_(event):
 
     def submit_current_answer(event):
         current_prompt = event.app.traverser.get_current_prompt()
-        prompt_buffer = event.app.layout.get_buffer_by_name(current_prompt)
-        event.app.values[current_prompt] = prompt_buffer.text
+        prompt_buffer = get_ui_control_by_buffer_name(
+            event.app.layout, current_prompt).buffer
+        event.app.traverser.submit_prompt_answer(prompt_buffer.text)
 
     @kb.add('tab')
     @kb.add('enter')
     def next_prompt(event):
         submit_current_answer(event)
         next_prompt = event.app.traverser.next_prompt()
-        event.app.layout.focus(next_prompt)
+        layout = event.app.layout
+        next_control = get_ui_control_by_buffer_name(layout, next_prompt)
+        layout.focus(next_control)
 
     @kb.add('s-tab')
```

Comment:
Why don't use constants from `prompt_toolkit.keys.Keys` instead

Reply:
I did it mainly because that is what the prompt toolkit documentation advertises, but I'm fine changing it to make it consistent with what auto-prompt mode does.